ref-farming/src/view.rs

```diff
@@ -263,4 +278,13 @@ impl Contract {
             })
             .collect()
     }
+
+    pub fn get_user_rps(&self, account_id: ValidAccountId, farm_id: FarmId) -> String {
+        let farmer = self.get_farmer(account_id.as_ref());
+        if let Some(rps) = farmer.get().user_rps.get(&farm_id) {
+            format!("{}", U256::from_little_endian(&rps))
+        } else {
+            String::from("")
```

Comment:
Should this be `"0"` instead?

Reply:
Yeah, maybe. Anyway, this view function is a little debug thought, frontend won't call it.